includes/class-wc-payments.php

```diff
@@ -347,6 +347,32 @@ public static function init() {
 
 		add_action( 'rest_api_init', [ __CLASS__, 'init_rest_api' ] );
 		add_action( 'woocommerce_woocommerce_payments_updated', [ __CLASS__, 'set_plugin_activation_timestamp' ] );
+
+		// Init Emails. TODO check if this is the right place to do it.
```

Comment:
Are there any other places in the plugin where we register custom emails?

Reply:
Need to check not sure

Reply:
I saw that other emails are added related to subscriptions. In the constructor of `WC_Payment_Gateway_WCPay` there is a call to `maybe_init_subscriptions`. This method is defined in `WC_Payment_Gateway_WCPay_Subscriptions_Trait` and among other things it adds the filter for `woocommerce_email_classes` (see [here](https://github.com/Automattic/woocommerce-payments/blob/2a789c3af62fdd412e357874686f9e1f33d50d31/includes/compat/subscriptions/trait-wc-payment-gateway-wcpay-subscriptions.php#L133))

Maybe I could follow a similar approach, after all it seems that the card readers are related to the Stripe gateway. What do you think @kalessil ?

Reply:
I tried to move the initialization of the email to the `WC_Payment_Gateway_WCPay`  but had to move it mack because the email was not showing under the Emails tab in the WooCommerce  settings. Maybe I am doing something wrong